webapi/src/test/scala/org/knora/webapi/slice/export/api/service/ExportServiceSpec.scala

```diff
@@ -0,0 +1,159 @@
+      name = "http://www.knora.org/data/admin",
+    ),
+  ).toSet
+
+  override def spec: Spec[TestEnvironment with Scope, Any] =
+    suite("ExportServiceSpec")(
+      test("basic") {
+        for {
+          _             <- ZIO.serviceWithZIO[TriplestoreService](_.insertDataIntoTriplestore(dataSets.toList, false))
+          _             <- ZIO.serviceWithZIO[OntologyCache](_.refreshCache())
+          project       <- ZIO.serviceWithZIO[KnoraProjectService](_.findById(projectIri)).map(_.get)
+          exportService <- ZIO.service[ExportService]
+          exportedCsv <-
+            exportService.exportResources(
+              project,
+              resourceClassIri,
+              List(
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#Place")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#LinkPropertyValue")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#TextParagraph")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#TextRich")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#FunkList")),
+              ),
+              user,
+              LanguageCode.EN,
+              includeIris = true,
+            )
+          csv <- exportService.toCsv(exportedCsv)
+        } yield assertGolden(csv, "basic")
+      },
+      test("with includeIris = false") {
+        for {
+          _             <- ZIO.serviceWithZIO[TriplestoreService](_.insertDataIntoTriplestore(dataSets.toList, false))
+          _             <- ZIO.serviceWithZIO[OntologyCache](_.refreshCache())
+          project       <- ZIO.serviceWithZIO[KnoraProjectService](_.findById(projectIri)).map(_.get)
+          exportService <- ZIO.service[ExportService]
+          exportedCsv <-
+            exportService.exportResources(
+              project,
+              resourceClassIri,
+              List(
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#Place")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#LinkPropertyValue")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#TextParagraph")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#TextRich")),
+                PropertyIri.unsafeFrom(sf.toSmartIri("http://www.knora.org/ontology/1612/Data#FunkList")),
+              ),
+              user,
+              LanguageCode.DE,
+              includeIris = false,
+            )
+          csv <- exportService.toCsv(exportedCsv)
+        } yield assertGolden(csv, "includeIrisFalse")
+      },
+    ).provide(
+      // almost all of these are for ReadResourcesService, which I would call a "little much", if I'm being prefectly honest
```

Comment:
If this is a real concern for you please use an E2EZSpec. Otherwise remove the comment please.

I myself have tried to write these kind of "unit" tests and felt that maintaining the list of layers in `.provide` in the long run is a real big problem when refactoring and breaking up the bigger "services" like the `ListsResponder`. IMHO the benefit of having a faster test is not worth this effort.